i18n/translations/fr/form-builder-responses.json

```
@@ -307,6 +307,10 @@
     "awatingConfirm": {
       "title": "État : En attente de confirmation",
       "description": "Les soumissions téléchargées peuvent être confirmées avec une demande API"
+    },
+    "overdue": {
+      "title": "Status: Overdue [FR]",
+      "description": "Some submissions are overdue for retrieval [FR]"
```

```suggestion
```suggestion
    },
    "overdue": {
      "title": "Réponses en retard",
      "description": "Récupérez et confirmez les soumissions avec l'API"
```
```